### DISCUSSION TITLE
Skin model molecule start values in MoBi 11

### DISCUSSION THREAD

**Discussion by Slow-Walker66** [Open-Systems-Pharmacology/Forum#1298] [Questions & Problems] [answered]
Hi, all

I had a problem with  the skin model courses. In the setting of "Molecule Start Values" chapter, it was said after filter the Molecule Name and Path Element 0, the next step should "Right-click on a non-functional grey aera of the graphical user interface and click 'STRG' and 'A' ". 
However, I cannot find the right place to click 'STRG' and 'A', resulted in the next simulation process run error with CYP1A2 patition coefficient (see the picture below).
Could you please help me to solve this problem?
Thank you in advance.
![guide of skin model development](https://user-images.githubusercontent.com/87760070/189367995-c473e7a5-71e4-4212-96b9-f1b6ced9139a.jpg)
<img width="1278" alt="Molecule start value panel" src="https://user-images.githubusercontent.com/87760070/189368001-328cdaa1-7766-42df-8df5-a4475aebe545.png">
![simulation error](https://user-images.githubusercontent.com/87760070/189368005-008105d0-b94d-42ae-a550-2e07e3d2f300.jpg)

**Comment by StephanSchaller** [comment]
Yes, to put it into context: this is from the esqLABS online training portal (https://learn-esqlabs.talentlms.com/). @maalbrec89, can you check? Thanks!

**Comment by maalbrec89** [comment, marked as answer]
Hello  @1998225 ,

I see in your screenshot that you use MoBi version 11. The dermal course is not yet updated to version 11, resulting in such experiences. We need some time to re-evaluate the courses and hope that we can release them within the next month. Thus, use only the OSP versions for which the course material has been published. 

In the course 'Starting with the OSP Suite' and the page 'Install the OSPS', you also find the mobile version of MoBi v10.

Here is a direct link
https://github.com/Open-Systems-Pharmacology/MoBi/releases/tag/v10.0.122


![GitHub](https://user-images.githubusercontent.com/52818721/189384262-7f8d2a64-74bb-4fd2-b517-4f4ea44b4adf.png)

**Reply by Slow-Walker66** [reply]
Sorry, I still could not find the 9th point (the course instruction) in Mobi 10, could you kindly provide me a picture to indicate where to click the 'STRG' and 'A' .
Thank you again.

**Reply by maalbrec89** [reply]
@1998225 

I have to apologize. STRG is the German Abbreviation for 'Steuerung', which is CTRL/Control in English. You might have to look for the Chinese equivalent on your keyboard. I corrected the material. Thank you and all the best.
![image](https://user-images.githubusercontent.com/52818721/189476590-d0390e09-97be-4983-8856-f3d24f34d33d.png)

**Reply by Slow-Walker66** [reply]
Thank you for your solution! It helps a lot!

**Comment by maalbrec89** [comment]
@PavelBal 
Notice the GUI issues:

![image](https://user-images.githubusercontent.com/52818721/189383235-2f41fe14-dbcf-42dd-b7bb-5285379d7ce8.png)
